src/templates/base/2019/base.html

```diff
@@ -278,6 +289,80 @@
           });
         }
 
+        function closeAnyOtherOpenDropdown(e) {
+          if (e.target.classList.contains('dropdown-open')) {
+            return
+          };
+          var openDropdownBtn = document.querySelector('.nav-dropdown-btn.dropdown-open');
+          openDropdownBtn && openDropdownBtn.click();
+        }
+        function trapFocusInList(e) {
+          var list = e.currentTarget;
+          var isInFooter = list.classList.contains('footer-list');
+          if(e.key === "ArrowDown") {
+            var siblingElem = isInFooter ? e.target.parentElement.previousElementSibling : e.target.parentElement.nextElementSibling;
+            var focusableElem = siblingElem ? siblingElem.querySelector('a') : (isInFooter ? lastFocusableElementInList : firstFocusableElementInList);
+            e.preventDefault();
+            focusableElem.focus();
+          } else if (e.key === "ArrowUp") {
```

Comment:
After expanding the dropdown list in the footer, I would expect the "Up" key to focus on the option directly above the button. It seems to be scrolling up instead. I need to press "Down" to focus on one of the options before being able to use the "Up" key in this way.